recipes/perl/meta.yaml

```diff
@@ -0,0 +1,25 @@
+package:
+  name: perl
+  version: 5.20.3.1
+
+source:
+  fn:  strawberry-perl-5.20.3.1-32bit-portable.zip [win]
```

Comment:
I was proposing to add a Perl recipe for Mac and Linux anyways (before I knew about this). Proposed changes to this PR can be seen here ( https://github.com/msarahan/staged-recipes/pull/1 ).